book/src/users/startup.md

```diff
@@ -48,22 +62,75 @@ every time Trin is restarted. The only exceptions are if...
 - User deletes the `TRIN_DATA_DIR` or changes the `TRIN_DATA_DIR`. In which 
   case a new private key will be randomly generated and used.
 
-### Sub-Protocols
-
-Trin can connect to different sub-protocols to have access to
-different types of data. One more more can be selected, but be aware
-that not all sub-protocols are ready:
-
-- Execution State Network
-- Execution History Network
-- Execution Transaction Gossip Network
-- Execution Canonical Indices Network
-
 ### Networking configuration
 
 Optionally one can specify Trin's network properties:
-- What sort of network connections (HTTP vs IPC)
+- What sort of connection to query with (HTTP vs IPC)
 - Port answering Ethereum-related queries
 - Port for connecting to other nodes
 
-These types of flags have defaults.
+## All flags
+
+Below is the current list of command line flags.
+
+Note that CLI flags may change over time, so run `trin --help` to get the most up-to-date information.
+
+```text
+Usage: trin [OPTIONS] [COMMAND]
+
+Commands:
+  create-dashboard  
+  help              Print this message or the help of the given subcommand(s)
+
+Options:
+      --web3-transport <WEB3_TRANSPORT>
```

Comment:
Oh one last thing I would prefer if this was it's own page so it is more findable named something like `Commandline options`. As normally I wouldn't expect these to be in the `startup` page, but that is just my preference we don't need to make that change in this PR.

Reply:
I feel mixed about it, but I'll try it this way. I liked the idea of having all of the flag-related content on one page. But we clearly have different preferences here, and the book should ideally serve both reading styles.

You want to be thrown into the deep end of reading all possible flags at once. I want an opinionated guide that cuts straight to the couple of flags that are most important to know about. So yeah, separate pages with a top-of-page links to the other is probably the best way to facilitate both wants.